_sass/blocks/how-it-works/_footer.scss

```diff
@@ -1,29 +1,36 @@
-.revenues_page-footer {
-  padding-bottom: $base-padding-large;
-  padding-top: $base-padding-large;
+.pre-footer {
```

Comment:
What is `pre-footer`, as in, what is the purpose of the component? I see it used a lot in this PR